lib/display.ex

```diff
@@ -21,9 +21,24 @@ defmodule Display do
   defp name(module), do: name(Atom.to_string(module))
 
   def show_failure(failure, module, name) do
+    intro(module, Tracker.visited)
```

Comment:
We only ever print failures. Success "roll on through" to the next koan.
I was torn between showing the intro in failure and adding a clause around display itself.

What if I rename it to `maybe_intro`? To sneaky? 😄